etl/data_helpers/geo.py

```diff
@@ -73,6 +82,10 @@ def list_countries_in_region(
 ) -> List[str]:
     """List countries that are members of a region.
 
+    ####################################################################################################################
+    WARNING: This function is deprecated, use list_members_of_region instead.
+    ####################################################################################################################
+
```

Comment:
I see that in the functions above, they have a similar section which is not included in the docstring, but here it is. We should probably include this in the docstring always are never (but not mix).

Reply:
Could use `warning.warn` to raise a `DeprecationWarning` message. That could be more useful.

```python
from warning import warn

def some_function():
    ...
    warn('This method is deprecated.', DeprecationWarning, stacklevel=2)
    ...
```

Reply:
Hey Lucas, I don't understand your first comment. The warnings in the `_load*` functions are not really necessary, since they are already internal functions, but I added so that we remember to not use them when creating functions in `geo` (and to signal that eventually we should remove them).
Having a deprecation warning implies that we would get many warnings when running old ETL steps.

Reply:
> Hey Lucas, I don't understand your first comment

In lines 67-69 (and others, e.g.58-60), the comment `WARNING: ...` is not included in the docstring. However, this one (85-87) is in the docstring.

To be consistent, I suggested having that comment either within or outside the docstring. However, I see that internal functions do not have a docstring; hence it is fine as it is.

> Having a deprecation warning implies that we would get many warnings when running old ETL steps.

I think that it is totally fine and actually positive to remind us that we are using deprecated functions. Also, we'd get such warnings if we use them accidentally now. So overall, it is a good strategy to prevent us from using these deprecated functions.

Reply:
For now I'd suggest to keep this in mind when reviewing PRs, and not add more warnings to the already long list of ETL unnecessary warnings. There's no rush to deprecate those functions given that we don't have good a good alternative for creating region aggregates (which is the most common goal when using these functions).

Reply:
Alright, maybe it is personal. I generally don't mind having error messages and actually appreciate them.

Without them, it becomes hard for me to track changes/recommendations like these. So not sure I'll "keep this in mind when reviewing PRs", will do my best to remember, though!

Reply:
Feel free to create a separate PR and raise a warning when these functions are used. I don't have a strong opinion on this.